src/main/java/de/tum/in/www1/artemis/service/ExerciseImportService.java

```diff
@@ -63,10 +59,6 @@ void copyExerciseBasis(final Exercise newExercise, final Exercise importedExerci
         }
     }
 
-    abstract Set<ExampleSubmission> copyExampleSubmission(final Exercise templateExercise, final Exercise newExercise);
```

Comment:
Pinging @akesfeden if it's okay to remove the methods from this generalized class.

Reply:
Is there a reason why this method and the one below is removed from here? As far as I can see, both implementations are still present. Although it seems like the override at `ModelingExerciseImportService` is not directly used, these abstract methods may be helpful for developing new subclasses as a guideline, that's why I asked.

Reply:
I think it would be nicer to keep them but I wouldn't insist on it so if you both think this is more understandable then it's okay for me

Reply:
Alright, I will leave it like this for now but I think this fits into the conversation of doing some refactoring/reworking of the import functionality so that more code is reused, also in the server but especially in the client.